specification/e-referrals-service-api.yaml

```diff
@@ -1279,19 +1350,20 @@ paths:
           schema:
             type: string
             example: '1,2'
-        - in: query
```

Comment:
is this no longer needed?

Reply:
Yes it is no longer required for the Story.